### PR TITLE
Fix invalid version parsing

### PR DESCRIPTION
**Pull request by dcermak**
cc @Vogtinator

### CODE REVIEW

File: obs-service-replace_using_package_version.spec [hunk 1]
```diff
@@ -31,6 +31,7 @@ Source2:        LICENSE
 BuildRequires:  sed
 Requires:       python3-docopt
 Requires:       python3-setuptools
```
Comment (davidcassany) [inline obs-service-replace_using_package_version.spec:33]: I am not sure this is needed any longer with such a change. IIRC this was only included for the `pkg_resource` import at runtime.

File: replace_using_package_version/replace_using_package_version.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 #
-# Copyright (c) 2018 SUSE Linux GmbH.  All rights reserved.
+# Copyright (c) SUSE LLC.  All rights reserved.
```
Comment (davidcassany) [inline replace_using_package_version/replace_using_package_version.py:4]: I think we should use everywhere

```
# Copyright (c) 2023 SUSE LLC
```

or

```
# Copyright (c) 2018 - 2023 SUSE LLC
```

File: hello-world-1.1.2.P3-1.x86_64.rpm
Comment (davidcassany) [inline hello-world-1.1.2.P3-1.x86_64.rpm:1]: Do we need that here at top level? Probably we could have an  `assets` folder inside `integration_tests`

**Review by davidcassany — COMMENTED** [review]
Thanks! The change looks good to me.

Beyond the obvious issue on tests failing to install/find python3-rpm I just added some nit comments on few small details.

File: obs-service-replace_using_package_version.spec (hunk 1 above)
Comment (dcermak) [reply obs-service-replace_using_package_version.spec:33]: Let's remove it, it shouldn't be needed.

File: replace_using_package_version/replace_using_package_version.py (hunk 2 above)
Comment (dcermak) [reply replace_using_package_version/replace_using_package_version.py:4]: Copyright years are actually not really necessary, but let's keep them to make legal happy.

**Comment by dcermak** [comment]
@davidcassany We have a nasty problem, as rpm is a system package, it's python bindings are only available to the system-default python… (see https://pypi.org/project/rpm/). This unfortunately breaks our test matrix with different python versions :-/

Do you think it would be sufficient that we rely on the integration tests here? They run on Leap, Tumbleweed and SLE, so we should get some decent test coverage via that.

**Comment by Vogtinator** [comment]
> @davidcassany We have a nasty problem, as rpm is a system package, it's python bindings are only available to the system-default python… (see https://pypi.org/project/rpm/). This unfortunately breaks our test matrix with different python versions :-/

There isn't really any reason why `python3-rpm.spec` couldn't be multipython FWICT, but for now that's what we have to work with...

**Comment by dcermak** [comment]
@davidcassany done. We will also have to adjust the branch protection settings (I can do that once we are all on board with this change)

**Pull request merged**